lib/utils.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+const assert = require('assert');
+const crypto = require('crypto');
+const Boom = require('boom');
+const bundleJS = require('@asset-pipe/js-reader');
+const bundleCSS = require('@asset-pipe/css-reader');
+const { promisify } = require('util');
+const body = promisify(require('body/json'));
+const schemas = require('./schemas');
+
+function createHashFromContent(content) {
+    const hasher = crypto.createHash('sha256');
+    hasher.update(content, 'utf8');
+    return hasher.digest('hex');
+}
+
+async function fetchFeeds(sink, ids) {
+    try {
+        return await Promise.all(ids.map(file => sink.get(file)));
+    } catch (err) {
+        throw Boom.boomify(err, {
+            statusCode: err.message.includes('No file with name') ? 404 : 500,
+            message: 'Unable to fetch 1 or more feeds from storage.',
+        });
+    }
+}
+
+function parseFeeds(rawFeeds) {
+    try {
+        return rawFeeds.map(JSON.parse);
```

Comment:
can we use https://github.com/zkat/json-parse-better-errors?

Reply:
yea sure!